lib/aruba/cucumber/hooks.rb

```diff
@@ -5,7 +5,7 @@
 
 if Aruba::VERSION >= '1.0.0'
   Around do |_, block|
-    Aruba.platform.with_environment(&block)
+    with_environment(&block)
```

Comment:
I did this change before and something was uterly broken afterwards - I was forced to revert that change. What's the reason for this change?